.github/workflows/test-lang-java.yml

```diff
@@ -90,28 +93,37 @@ jobs:
           restore-keys: |
             ${{ runner.os }}-maven-
 
-      - name: Setup Java
-        uses: actions/setup-java@v4
+      - name: 'Setup Temurin JDK 8, 11, 17 & 21'
+        uses: actions/setup-java@387ac29b308b003ca37ba93a6cab5eb57c8f5f93 # v4.0.0
         with:
           distribution: 'temurin'
-          java-version: ${{ matrix.java }}
+          java-version: |
+            8
+            11
+            17
+            21
+
+      - name: 'Setup Maven 3.9.6'
+        uses: stCarolas/setup-maven@07fbbe97d97ef44336b7382563d66743297e442f # v4.5
+        with:
+          maven-version: 3.9.6
 
-      - name: Setup Python for Generating Input Data
-        uses: actions/setup-python@v5
+      - name: 'Setup Python for Generating Input Data'
+        uses: actions/setup-python@0a5c61591373683505ea898e09a3ea4f39ef2b9c # v5.0.0
 
-      - name: Apt Install Compression Libs Required by Python
+      - name: 'Apt Install Compression Libs Required by Python'
```

Comment:
The quotes are not really needed

Reply:
I know, but I really want all code to look consistent and readable. And by having these quotes the syntax highlighting in my IDE (IntelliJ) makes it visually better to read (different colors).